Library/Homebrew/download_strategy.rb

```diff
@@ -1347,16 +1384,18 @@ def initialize(url, name, version, **meta)
   # @api public
   sig { returns(Time) }
   def source_modified_time
-    out, = silent_command("fossil", args: ["info", "tip", "-R", cached_location])
-    Time.parse(out[/^uuid: +\h+ (.+)$/, 1])
+    out = silent_command("fossil", args: ["info", "tip", "-R", cached_location]).stdout
+    time = out[/^uuid: +\h+ (.+)$/, 1]
+    Time.parse T.must(time)
```

Comment:
@MikeMcQuaid Not sure whether it'd be better to annotate is as `time = T.must(out[/^uuid: +\h+ (.+)$/, 1])`.

Reply:
@Kentzo Please split this into a separate PR too.